src/cmd/flux-account.py

```diff
@@ -293,6 +293,13 @@ def add_view_job_records_arg(subparsers):
         help="bank",
         metavar="BANK",
     )
+    subparser_view_job_records.add_argument(
+        "--fields",
+        type=str,
+        help="list of fields to include in output",
+        default="userid,username,jobid,t_submit,t_run,t_inactive,nnodes,project,bank",
+        metavar="USERID,USERNAME,JOBID,T_SUBMIT,T_RUN,T_INACTIVE,NNODES,PROJECT,BANK",
```

Comment:
Does having this long `metavar` not make the help/usage message kind of weird looking?

Reply:
I don't think so? But maybe it's just my terminal?

```console
usage: flux-account.py view-job-records [-h] [-u USERNAME] [-j JOBID]
                                        [-a START TIME] [-b END TIME]
                                        [--project PROJECT] [--bank BANK]
                                        [--fields USERID,USERNAME,JOBID,T_SUBMIT,T_RUN,T_INACTIVE,NNODES,PROJECT,BANK]
```

I could probably just break this into a multi-line string though in `flux-account.py`. FWIW, it's less than 90 characters which I think is compatible with `black` formatting.